api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md

```diff
@@ -1,5 +1,5 @@
 ---
 ##### shortDescription
-<!-- Description goes here -->
+This section describes methods that control the **FileUploader** widget.
```

Comment:
What do you mean by "that control"? Control it how?

Reply:
It can by any actions. Not the file uploader has only upload methods.

Reply:
You should add more information

Reply:
All 'Methods' topics in the DevExtreme documentation has the same description. 

There is no need to list all functionality as this description is followed by all widget's methods at the same page in the documentation.

Reply:
Ok, but it does not explain what you mean when you say it 'controls' the widget. What does 'control' imply? How does it control the widget? What does it do?

Reply:
It means that methods allow you to manage the widget. May I use:

> This section describes methods that manage the **FileUploader** widget.

Reply:
'manage', like 'control', do not explain what these methods do. How do they manage or control the widget? What do these methods do? If you say, "John manages a grocery store", you can ask, okay, what does he actually do? He supervises the staff in the store, he arranges deliveries of new goods, he ensures the accounting records are accurate, etc.